source/import-export.txt

```diff
@@ -49,20 +49,33 @@ that it is formatted correctly.
        name: JSON
        content: |
 
-         When importing data from a **JSON** file, each document must exist on
-         its own line in the file. Do not use commas at the end of lines to
-         separate documents.
+         When importing data from a **JSON** file, you can format your 
+         data as:
+
+         - Newline-delimited documents, or
+
+         - Comma-separated documents in an array
 
          .. example::
 
-            The following ``.json`` file imports three documents:
+            The following newline-delimited ``.json`` file is formatted 
+            correctly:
 
             .. code-block:: javascript
 
                { "type": "home", "number": "212-555-1234" }
                { "type": "cell", "number": "646-555-4567" }
                { "type": "office", "number": "202-555-0182"}
 
+            The following comma-separated ``.json`` array file is also 
+            formatted correctly:
+
+            .. code-block:: javascript
+
+               [{ "type": "home", "number": "212-555-1234" }, { "type": "cell", "number": "646-555-4567" }, { "type": "office", "number": "202-555-0182"}]
+
+            |compass-short| ignores line breaks in JSON arrays.
```

Comment:
[super nit] Can we make the following paragraph |compass-short| instead of |compass| to keep them parallel?

Reply:
Done, and went with a happy medium of 70% for the images.